guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
+
+#### Guild#create
+
+`Guild#systemChannelFlags` can now be set in the `Guild#create` method.
+
+#### Guild#edit
+
+The `Guild#description` and `Guild#features` properties can now be edited.
+
+#### Guild#editWelcomeScreen
+
+Provides API support for bots to edit the Guild's `WelcomeScreen`.
+
+#### Guild#emojis
+
+The `GuildEmojiManager` class now extends `BaseGuildEmojiManager`.
+
+In addition to the existing methods, it now supports `GuildEmojiManager#fetch`.
+
+#### Guild#fetchWelcomeScreen
+
+Provides API support for fetching the Guild's `WelcomeScreen`.
+
+#### Guild#fetchWidget
+
+Provides API support for the Guild's Widget, containing information about the guild and it's members.
+
+#### Guild#invites
+
+Provides access to the new `GuildInviteManager`.
+
+#### Guild#nsfwLevel
+
+Guilds can now be marked as NSFW.
+
+#### Guild#owner
+
+The `Guild#owner` property has been removed as it was unreliable due to caching, replaced with `Guild#fetchOwner`.
+
+```diff
+- console.log(guild.owner);
++ guild.fetchOwner().then(console.log);
+```
+
+#### Guild#premiumTier
+
+The `Guild#premiumTier` property is now represented by the `PremiumTier` enum.
+
+#### Guild#setChannelPositions
+
+Now supports setting the parent of multiple channels, and locking their permissions via the `ChannelPosition#parent` and `ChannelPosition#lockPermissions` options.
+
+### GuildBanManager
+
+Provides improved API support for handling and caching bans.
+
+### GuildChannel
+
+#### GuildChannel#clone
+
+Now supports setting the `position` property.
+
+#### GuildChannel#createInvite
+
+Now supports additional options:
+
+- `targetUser` to target the invite to join a particular streaming user
+- `targetApplication` to target the invite to a particular Discord activity
+- `targetType`
+
+### GuildChannelManager
+
+#### GuildChannelManager#fetch
+
+Now supports fetching the channels of a Guild.
+
+### GuildInviteManager
+
+Aligns support for creating and fetching invites with the managers design.
+
+Replaces `Guild#fetchInvites`.
+
+### GuildManager
+
+#### GuildManager#create
+
+Now supports specifying the AFK and system channels when creating a new guild.
+
+#### GuildManager#fetch
+
+Now supports fetching multiple guilds, returning a `Promise<Collection<Snowflake, OAuth2Guild>>` if used in this way.
+
+### GuildEmojiManager
+
+#### GuildEmojiManager#fetch
+
+Provides API support for the `GET /guilds/{guild.id}/emojis` endpoint.
+
+### GuildMember
+
+#### GuildMember#pending
+
+Flags whether a member has passed the guild's membership gate.
+
+The flag is `true` before accepting and fires `guildMemberUpdate` when the member accepts.
+
+### GuildMemberManager
+
+Several methods were added to `GuildMemberManager` to provide API support for uncached members.
+
+#### GuildMemberManager#edit
+
+`guild.members.edit('123456789012345678', data, reason)`
+
+Approximately equivalent to `GuildMember#edit(data, reason)` but does not resolve to a GuildMember.
```

Comment:
This returns a Promise with a GuildMember so this should probably be changed

Reply:
So,
```suggestion
Equivalent to `GuildMember#edit(data, reason)`.
```

Reply:
Sounds about right yeah